documentation/contributing.md

```diff
@@ -0,0 +1,91 @@
+# Development
```

Comment:
Personal preference but I think `CONTRIBUTING.md` should generally live in the project root

Reply:
💯 that mstm